packages/server/lib/browsers/chrome.ts

```diff
@@ -700,10 +700,15 @@ export = {
     // first allows us to connect the remote interface,
     // start video recording and then
     // we will load the actual page
-    const launchedBrowser = await launch(browser, 'about:blank', port, args, launchOptions.env) as unknown as BrowserInstance & { browserCriClient: BrowserCriClient}
+    const process = launch(browser, 'about:blank', port, args, launchOptions.env)
```

Comment:
doesn't this still need to be awaited? Why the change here?

Reply:
> doesn't this still need to be awaited? Why the change here?

The launch function doesn't return a promise. There was a stub that was async (which I changed to non-async) but the actual implementation wasn't. The Firefox implementation called launch without await so I dropped it in the Chrome implementation as well.